script/dump_stats.rb

```diff
@@ -0,0 +1,107 @@
+# 1. Run this script on production as follows:
+#      RAILS_ENV=production bundle exec rails r script/dump_stats.rb
+# frozen_string_literal: true
+
+work_output = File.open(ENV['WORKS_FILENAME'] || '/tmp/works.csv', 'w')
+work_output.puts "Created_at,language,restricted,complete,word_count,tags\n"
+if ENV['QUICK'].nil?
+  stats_output = File.open(ENV['STATS_FILENAME'] || '/tmp/stats.csv', 'w')
+  stats_output.puts "count_of_days,to,from,day_of_week,tag_type,tag_ids,count"
+end
+first_time = Work.first.created_at
+days = 1
+to = DateTime.now.beginning_of_day
```

Comment:
The `to` and `from` keep tripping me up... would `end_date` and `start_date` be accurate variable names?